From 7a04c83f373a40079c0ca2f16ffe96921e2b9608 Mon Sep 17 00:00:00 2001
From: Niels Breet <niels.breet@jolla.com>
Date: Wed, 27 May 2020 17:23:00 +0300
Subject: [PATCH] Use GNUInstallDirs

---
 CMakeLists.txt | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c2ce786..0471903 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -53,12 +53,14 @@ ENDIF( WBXML_INSTALL_FULL_HEADERS )
 
 CMAKE_MINIMUM_REQUIRED(VERSION 2.4)
 
+include(GNUInstallDirs)
+
 # TODO: Move to external file/macro
 SET( CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules" )
 
 SET( LIB_SUFFIX "" CACHE STRING "The library directory suffix. 32bit empty string, 64 for 64bit." )
-SET( LIB_INSTALL_DIR  "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}" CACHE INTERNAL "libary location" )
-SET( LIBDATA_INSTALL_DIR  "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}" CACHE PATH "The library data directory" )
+SET( LIB_INSTALL_DIR  "${CMAKE_INSTALL_FULL_LIBDIR}" CACHE INTERNAL "libary location" )
+SET( LIBDATA_INSTALL_DIR  "${CMAKE_INSTALL_FULL_LIBDIR}" CACHE PATH "The library data directory" )
 SET( BIN_INSTALL_DIR  "${CMAKE_INSTALL_PREFIX}/bin" CACHE INTERNAL "binary location" )
 SET( SHARE_INSTALL_DIR  "${CMAKE_INSTALL_PREFIX}/share" CACHE INTERNAL "data location" )
 SET( INCLUDE_INSTALL_DIR  "${CMAKE_INSTALL_PREFIX}/include" CACHE INTERNAL "headers location" )
-- 
1.9.1

